recipes/fast-histogram/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fast-histogram" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "1cbefcb23a8fad6f904b73a6a8d7cc04127e3691cbb6f0cdd4fdd20a94f0dd0c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy
```

Comment:
I think this should be `numpy x.x` here in the `build` section and also below in the `run` section since the C-extension is compiled against the numpy C-API. See:

https://github.com/conda-forge/staged-recipes#4-what-does-numpy-xx-mean